package.json

```diff
@@ -1,12 +1,29 @@
 {
+  "name": "react-webpack-setup",
```

Comment:
That's not your repo name!

Reply:
🍻

Reply:
No need to shout...Maybe I want it to be my repo name!....................